src/main/java/uk/nhs/hee/tis/revalidation/migration/reader/BatchDataReader.java

```diff
@@ -82,14 +82,14 @@ public Job job(JobBuilderFactory jobBuilderFactory,
   private PagingQueryProvider createQuery() {
     MySqlPagingQueryProvider queryProvider = new MySqlPagingQueryProvider();
     queryProvider.setSelectClause("SELECT * ");
-    queryProvider.setFromClause("FROM Revalidation");
+    queryProvider.setFromClause("FROM revalidation.Revalidation reval INNER JOIN tcs.GmcDetails gmc ON reval.tisId = gmc.id");
```

Comment:
Wondering do we want a LEFT JOIN instead of INNER JOIN to make sure all the data in the Revalidation table can be migrated to the new Reval?